screens/Issuers/IssuersScreen.tsx

```diff
@@ -124,24 +129,56 @@ export const IssuersScreen: React.FC<
     );
   }
 
+  const showAllIssuers = () => {
+    setFilteredSearchData(issuers);
+  };
+
+  const searchIssuer = text => {
+    if (text === '') {
```

Comment:
Instead of if-else can we add reframe it this way.

let filterdData = issuers;
if (text.length > 1) {
      filterdData = issuers.filter(item => {
        if (
          getDisplayObjectForCurrentLanguage(item.display)
            ?.name.toLowerCase()
            .includes(text.toLowerCase())
        ) {
          return getDisplayObjectForCurrentLanguage(item.display);
        }
      }, []);
    }
setFilteredSearchData(filterdData);
setSearch(text);